controller.go

```diff
@@ -424,7 +424,7 @@ func jobSpec(apprepo *apprepov1alpha1.AppRepository) batchv1.JobSpec {
 					{
 						Name:    "sync",
 						Image:   repoSyncImage,
```

Comment:
I think we probably want to also change the docker repo name to `kubeapps/chart-repo` or something. I can push @ngtuna's version there.

Reply:
Sure. Please push the image and let me know, I can actually test end-to-end with proper image.

Reply:
Sorry, got distracted. Just pushed kubeapps/chart-repo:latest :)